src/main/scala/rocketchip/BaseTop.scala

```diff
@@ -67,7 +67,8 @@ abstract class BaseTopModule[+L <: BaseTop, +B <: BaseTopBundle](
   val io: B = b
 
   val coreplex = p(BuildCoreplex)(outer.c, p)
-  val coreplexIO = coreplex.io
+  val coreplexIO = Wire(new BaseCoreplexBundle(outer.c)(p){})
```

Comment:
This is bad, since if the user extended the type, you won't get the same wire.  You're better off doing "val coreplexIO = Wire(coreplex.io)" and implicitly calling the clone method.